sdk/monitor/Azure.ResourceManager.Monitor/src/Customized/Models/MonitorScaleCapacity.Serialization.cs

```diff
@@ -8,48 +8,50 @@
 using System.Text.Json;
 using Azure.Core;
 
-[assembly: CodeGenSuppressType("MonitorScaleCapacity")]
-
 namespace Azure.ResourceManager.Monitor.Models
 {
     public partial class MonitorScaleCapacity : IUtf8JsonSerializable
     {
-        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        void global::Azure.Core.IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
```

Comment:
nit: we can remove the global::...

Reply:
This is known issue when we want to use customization code to override this explicitly implemented method: https://github.com/Azure/autorest.csharp/issues/3214

Reply:
Same thing happens when you want to suppress it using CodeGenSuppress.